docs/.vuepress/enhanceApp.js

```diff
@@ -1,9 +1,9 @@
 export default ({ router }) => {
   router.addRoutes([
-    { path: '/main/spec/*', redirect: '/main/modules/' },
-    { path: '/main/spec/governance/', redirect: '/main/modules/gov/' },
-    { path: '/v0.41/', redirect: '/v0.42/' },
-    { path: '/v0.43/', redirect: '/v0.44/' },
-    { path: '/master/', redirect: '/main/' },
+    { path: '/main/spec/*', redirect: '/modules/' },
+    { path: '/main/spec/governance/', redirect: '/modules/gov/' },
+    { path: '/v0.41/', redirect: '/v0.42/' }, // TODO to fix
+    { path: '/v0.43/', redirect: '/v0.44/' }, // TODO to fix
```

Comment:
Can we avoid naked TODOs and reference an issue?

Reply:
Sure, added!